doc/supply_demand.rst

```diff
@@ -169,6 +169,77 @@ Activated in Config from the `tes <https://github.com/PyPSA/pypsa-eur-sec/blob/3
+
+*Time windows and constraints*
+
+Heat demand can be shifted within configurable time periods to avoid buildings acting as long-term seasonal storage. By default, 12-hour periods are used:
+
+- **Day period**: 9am to 9pm (21:00)
+- **Night period**: 9pm (21:00) to 9am
+
+At the boundaries between these periods (configured via `residential_heat_restriction_time <https://github.com/PyPSA/pypsa-eur/blob/master/config/config.default.yaml>`__, default: [10, 22] corresponding to 9am and 9pm local time), the thermal storage state of charge must return to its baseline level. The implementation automatically adjusts these checkpoint hours to each country's local timezone, ensuring consistent behavioral patterns across different regions. This ensures that heat consumption requirements are met within each time window while allowing temporal load shifting for demand response. Users can adjust the checkpoint hours to create different period lengths as needed.
+
+*Storage capacity and flexibility magnitude*
+
+The flexibility storage capacity is sized based on the maximum residential space heating demand at each node. The actual available flexibility is constrained by the `residential_heat_restriction_value <https://github.com/PyPSA/pypsa-eur/blob/master/config/config.default.yaml>`__ parameter (default: 0.27), which sets the maximum state of charge as a fraction of the storage capacity.
+
+The default value of 0.27 represents a conservative assumption for the thermal buffer capacity available in residential buildings, balancing flexibility potential against thermal comfort constraints. This parameter is user-configurable and can be adjusted based on specific assumptions about building thermal mass, insulation quality, and acceptable temperature variations. Setting it to 0 disables heat flexibility, while values closer to 1.0 allow greater load shifting (though this may require stronger assumptions about building characteristics and occupant comfort tolerance).
+
+*Thermal characteristics*
+
+Thermal losses are modeled using the same standing loss rates as decentralized water tank storage (a simplifying assumption that likely underestimates actual building losses). The storage is cyclic, meaning the state of charge at the end of the optimization period must equal that at the beginning.
```

Comment:
I would expect standing losses of buildings to be lower than that of a water tank. This is perhaps something to get a feeling for, but not a showstopper for merging.